### PR TITLE
fix(SEC): Fixing CVE-2021-31525

### PR DESCRIPTION
**Pull request by sanjayk-github-dev**
## Problem
 
CVE-2021-31525
## Solution
 
Fixing CVE-2021-31525 by bumping version

### CODE REVIEW

**Review by skrdgraph — APPROVED** [review]
this will probably fix other CVEs as well.

File: go.mod [hunk 1]
```diff
@@ -62,10 +62,10 @@ require (
 	go.opencensus.io v0.22.5
 	go.uber.org/zap v1.16.0
 	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
-	golang.org/x/net v0.0.0-20201021035429-f5854403a974
+	golang.org/x/net v0.0.0-20210428140749-89ef3d95e781
```
Comment (sanjayk-github-dev) [inline go.mod:65]: @skrdgraph I changed this to address CVE-2021-31525. However `go mod tidy` made additional changes.

Comment (skrdgraph) [reply go.mod:65]: yes this is true, the current sec-CVE count is at 70 (before merging this PR). The other fixes that came in through `go mod tidy` will be addressing other CVEs for sure. We can look at the CVE count after this merge. We are expecting the number to drop to 69, but if it goes further below - it simply means we have fixed more CVEs with this single PR.

**Review by meghalims — APPROVED** [review]
Looks good 👍 .
Please sign the CLA

**Comment by sanjayk-github-dev** [comment]
> Looks good 👍 . Please sign the CLA

@meghalims I have signed the CLA, but it takes a day to reflect per @skrdgraph .

**Comment by meghalims** [comment]
> > Looks good 👍 . Please sign the CLA
> 
> @meghalims I have signed the CLA, but it takes a day to reflect per @skrdgraph .

Ahh yes. Thanks Sanjay

**Pull request merged**